update_scripts/update_user_samples.py

```diff
@@ -26,7 +27,9 @@ def update(self):
         and process chained requests that these requests are members of
         """
         # Iterate through tags and add or update requests
-        table, entries_table = self.get_table_names()
+        table, entries_table, campaings_table = self.get_table_names_()
+        campaign_names = query(self.conn, campaings_table, ['name'])
+        campaign_names = [v.split('*')[0] for d in campaign_names for k, v in d.items()]
```

Comment:
Hm, this makes an assumption that all campaigns (GEN, Mini, Nano) have identical substrings in the front. While it is true based on the PdmV policy, it's a risky assumption to take.